password-reset-sql.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
This script should be replaced by an entry in upgrade.php and on create.sql: when this feature is accepted will be integral part of MailWatch, and should not have separate install instrunction